assets/sass/v2/_enroll-phone.scss

```diff
@@ -0,0 +1,35 @@
+.phone-authenticator-enroll {
+  .okta-form-subtitle {
+    text-align: center;
+  }
+  .chzn-container {
+    // TODO: Find a way around important
+    /* stylelint-disable declaration-no-important */
```

Comment:
Couldn't find a way to avoid `!important`since the library for the dropdown puts a width on it `inline` on the element. Please let me know if you know of a better way. (I'd rather keep it to CSS than in JS for this).